### ISSUE TITLE
pip --user install 

### ISSUE THREAD

**Issue by darked89** [jordanlab/tagore#6]
Hello,

I did:

```
pip install tagore --user
```

This puts tagore in ```~/.local/bin```.

Running it I got a following error:
```
Could not open /usr/lib/tagore-data/base.svg.p.  Please reinstall tagore
Traceback (most recent call last):
  File "/home/darked89/.local/bin/tagore", line 242, in <module>
    raise base_e
  File "/home/darked89/.local/bin/tagore", line 239, in <module>
    BASE = open(BASE_PATH, 'rb')
FileNotFoundError: [Errno 2] No such file or directory: '/usr/lib/tagore-data/base.svg.p'
```
But I do have the missing file in my home dir installation:
```$HOME/.local/lib/tagore-data/base.svg.p```

IMHO it is a bug since not on all systems we have root/sudo and moving bed files back and forth can be quite cumbersome. 

Hope it helps,

Darek Kedra

**EDIT**
BTW, the system wide install puts the file in question here: ```/usr/local/lib/tagore-data/base.svg.p```

**Comment by m-makarious** [comment]
Having this same issue, unfortunately - any updates?

**Comment by ar0ch** [comment]
@m-makarious @darked89 better late than never, just released 1.1.2 which should solve this
